### DISCUSSION TITLE
Recommended data provider when using external auth

### DISCUSSION THREAD

**Discussion by sergejostir** [drakkan/sftpgo#265] [Q&A] [answered]
What is the recommended data provider when using the external auth? I don't need any of the data to be preserved between sftpgo restarts.

I was thinking about using `memory` or `sqlite's :memory:`, but would maybe using normal sqlite (file bassed) be better, so not all data has to be preserved in the memory? Were there maybe any tests made about the sftpgo's memory consumption?

**Comment by drakkan** [comment, marked as answer]
Hi,

if you don't have a lot of users probably memory provider is the best one. I haven't done specific tests but I believe memory usage will be low unless you have thousands of users
